init_tables.sql

```diff
@@ -20,8 +20,8 @@ CREATE TABLE playlist_tracks (
     playlist_id integer NOT NULL,
     song_id integer NOT NULL,
     PRIMARY KEY (id),
-    FOREIGN KEY (playlist_id) REFERENCES playlists(id),
-    FOREIGN KEY (song_id) REFERENCES tracks(id)
+    FOREIGN KEY (playlist_id) REFERENCES playlists(id)
+    --FOREIGN KEY (song_id) REFERENCES tracks(id) tracks doesnt exist
```

Comment:
Not sure why you commented this line. You mention that tracks doesn't exist but there is a tracks table...